draft-spaghetti-sidrops-rpki-ta-tiebreaker.xml

```diff
@@ -172,12 +173,20 @@ version="3">
         </li>
       </ul>
     </section>
+    <section>
+      <name>Tiebreaker design</name>
+      <t>
+        The tiebreaker scheme in this document aims to create a partial order over TA certificates issued by the same TA.
+        The goal of this order is to, in all realistic scenarios, allow the TA to always issue a certificate that is preferred over any previous certificate.
+      </t>
+    </section>
 
     <section anchor="security">
       <name>Security Considerations</name>
       <t>
         When Relying Parties inadvertently use a different instance of the TA certificate than the TA operator intended for RPs to use, the certification path validation process will yield an unexpected outcome.
         Standardization of a tiebreaking scheme helps both RP and TA operators arrive at deterministic outcomes.
+        The proposed tiebreaking scheme prevents RPs from accepting a previous certificate presented by an on-path attacker, which may cause validation failures.
```

Comment:
It's not just validation failures, it's also validating things that should not be validated if at some point the TA changes (reduces) the INRs in the TA cert. I think it's already in the first sentence of the section, so the addition of ', which may cause validation failures' can probably be dropped.

Reply:
The TALs have all issued 0/0 certificates, so a persistent on-path attacker can prevent any future reduction of the resource set.

I moved up the cases until after the first sentence.